packages/oidc/src/dpop/dpop.spec.ts

```diff
@@ -218,4 +218,25 @@ describe("validateIdToken", () => {
       )
     ).resolves.toBe(true);
   });
+
+  it("returns true if the ID token is valid for an RSA signature", async () => {
+    const payload = {
+      sub: "https://my.webid",
+      iss: "https://some.issuer",
+      aud: "https://my.app/registration",
+      iat: 2551294137,
+      exp: 2551301337,
+    };
+    const idToken = await signJwt(payload, mockJwk(), {
+      algorithm: "RS256",
```

Comment:
Are you planning to also support ES256 at some point?

Reply:
Currently, ES256 is the algorithm we use by default for DPoP tokens, and both ES256 and RS256 are supported for ID token validation. I was under the understanding that verifying EC signatures was faster than RSA, wich is why it was more convenient for DPoP tokens, where the signature is verified on each request. However, I don't remember at all on which source I'm basing this, so it may very well be an error from me, and I'm happy to talk about it to learn more.

Reply:
Sounds great. In my experience, ES256 signatures are faster to transfer, verify and compute, though not by much. The real difference is in key generation, where ECDSA key pairs are much, much faster to create than RSA key pairs.